### PR TITLE
[Windcentrale] Use DS annotations, fix static code analysis findings

### PR DESCRIPTION
**Pull request by wborn**
Polish for the Windcentrale binding:
* Use DS annotations
* Fix static code analysis findings

### CODE REVIEW

File: addons/binding/org.openhab.binding.windcentrale/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -25,5 +25,7 @@ Import-Package:
  org.eclipse.smarthome.io.net.http,
  org.openhab.binding.windcentrale,
  org.openhab.binding.windcentrale.handler,
+ org.osgi.framework,
+ org.osgi.service.component.annotations;resolution:=optional,
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.windcentrale/META-INF/MANIFEST.MF]: you can remove this line again - it was a bug in the parent pom that this was needed yesterday. But it has been fixed with https://github.com/openhab/openhab-core/pull/237

Comment (wborn) [reply addons/binding/org.openhab.binding.windcentrale/META-INF/MANIFEST.MF]: There are a lot of other bindings with the same line in their manifest. Does that mean a PR should be created to remove it from all those as well?

**Comment by kaikreuzer** [comment]
If you see this anywhere else, feel free to remove it!

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**